libs/pika/resource_partitioner/examples/oversubscribing_resource_partitioner.cpp

```diff
@@ -235,7 +235,8 @@ void init_resource_partitioner_handler(pika::resource::partitioner& rp,
     {
         // we use unspecified as the scheduler type and it will be set according to
         // the --pika:queuing=xxx option or default.
-        std::uint32_t deft = pika::threads::scheduler_mode::default_mode;
+        std::uint32_t deft = static_cast<std::uint32_t>(
+            pika::threads::scheduler_mode::default_mode);
         rp.create_thread_pool(pool_name,
             pika::resource::scheduling_policy::shared_priority,
             pika::threads::scheduler_mode(deft));
```

Comment:
```suggestion
        auto deft = pika::threads::scheduler_mode::default_mode;
        rp.create_thread_pool(pool_name,
            pika::resource::scheduling_policy::shared_priority,
            deft);
```
and the same in `simple_resource_partitioner.cpp`.